integration-tests/smoke/ccip/ccip_messaging_test.go

```diff
@@ -429,6 +431,13 @@ func Test_CCIPMessaging_EVM2Solana(t *testing.T) {
 		require.NoError(t, err, "failed to get account info")
 		require.Equal(t, uint8(0), receiverCounterAccount.Value)
 
+		deployer := solChains[sourceChain].DeployerKey
```

Comment:
I think this is what you intended
```suggestion
		deployer := solChains[destChain].DeployerKey
```